assets/CommonAssemblyInfo.cs

```diff
@@ -1,5 +1,5 @@
 using System.Reflection;
 
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.1.1.1")]
+[assembly: AssemblyFileVersion("1.4.204.0")]
```

Comment:
Hmmm, I changed this line manually so it matches Serilog assembly version.
How do I handle this file properly? Just don't alter?